src/ocean/LibVersion.d

```diff
@@ -0,0 +1,9 @@
+module ocean.LibVersion;
+
+const has_features_2_6 = true;
```

Comment:
But again, you can't rely on this feature because then if you introduce this in v2.5.7, anyone still at v2.5.6 or lower will have the check failing. I know people is supposed to update to the latest patch release, but it won't always happen, and if you can't rely on this feature `const` being correctly defined, it means it's not really necessary to add the check at this point, so why add it?

For me is best and cleaner to do a clean cut and support this idiom only for v2.7+